Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -352,6 +352,12 @@ protected function initialize()
             }
         }
 
+        foreach ($this->settings['reflection']['ignoredNamespaces'] as $namespace => $ignoreFlag) {
```

Comment:
Yeah, we should also cover that with a test as well! I'll take care! Wasn't aware of that `isTagIgnored()` Method. What about a new `isNamespaceIgnored()` @albe ?